server/exception_code.py

```diff
@@ -0,0 +1 @@
+GAMEIDERROR = 1
```

Comment:
`GAMEID_ERROR = 'GAMEID_ERROR'` will be better. A number says nothing about the error

Reply:
done